packages/developer-console-spa/src/components/ConfigureSearch/SearchWizardForm.tsx

```diff
@@ -0,0 +1,186 @@
+import {
+  Button,
+  Card,
+  CardBody,
+  Flex,
+  FlexItem,
+  Stack,
+  StackItem,
+  Text,
+} from '@patternfly/react-core';
+import { createStore, useStateMachine } from 'little-state-machine';
+import { useCallback, useContext, useEffect, useMemo, useState } from 'react';
+import { updateSearchConfigService } from 'services/searchConfig';
+import { AppContext } from '../../context/AppContext';
+import useSearchConfig from '../../hooks/useSearchConfig';
+import Header from '../Header';
+import Loader from '../Loader';
+import ConfigureSearchStep1 from './Steps/Step1';
+import ConfigureSearchStep2 from './Steps/Step2';
+import ConfigureSearchStep3 from './Steps/Step3';
+import saveState, { overrideState } from './Steps/store';
+import { clamp, isEmpty } from 'lodash';
+import { CSSProperties } from 'react';
```

Comment:
Please join this one with top react import

Reply:
Updated.